packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1091,8 +1091,6 @@ export class DeltaManager
         this._outbound.clear();
         this.emit("disconnect", reason);
 
-        // Avoid re-entrancy - remove all listeners before closing!
-        connection.removeAllListeners();
```

Comment:
This does not look right - events will get raised in close() and our handlers will get called.
We may get into reconnectOnError() and it will assert

Reply:
Yeah, I'm looking through it now and will update soon.